### ISSUE TITLE
count elements

### ISSUE THREAD

**Issue by brownman** [MobileCaddy/protractor-ionic-locator#1]
always 20 rows

**Comment by toddhalfpenny** [comment]
Agreed, this is an issue.
However this actually comes from the number of DOM elements actually created by Ionic itself.
This [post](http://developer.mobilecaddy.net/2015/05/end2end-testing-ionic-collection-repeat-with-protractor/) describes it (and work-arounds) and we have raised a [PR on the Ionic repo](https://github.com/driftyco/ionic/pull/3830) itself to cover this. Perhaps a +1 on that PR might help move things along.

My suggestion is to keep this issue open so that folk can see it, but not to fix it within this project.

**Comment by Simpler1** [comment]
So this package is useless in it's current form.  Right?

The test included with this package creates an array of 6 elements and then "successfully" tests to see if there are 20.

**Comment by toddhalfpenny** [comment]
No, not useless. As a locator it works fine (see the example in the README), except for the the `count` function. This however, as mentioned above is down to the Ionic implementation, of which a PR was opened to address.

**Comment by Simpler1** [comment]
I appreciate the effort and really wish this worked, but if the count is wrong, then it array doesn't contain all of the elements from the repeat.

Thanks again...
